sources/VT100Terminal.m

```diff
@@ -2498,6 +2498,12 @@ - (void)executeXtermSetPalette:(VT100Token *)token {
 
 - (void)executeXtermProprietaryExtermExtension:(VT100Token *)token {
     NSString* argument = token.string;
+    if ([argument hasPrefix:@"file://"]) {  // Support for proxy icon, following Terminal.app syntax
```

Comment:
I think it's better to take this route if the argument does not begin with a digit. That way empty string can reset it.